packages/metapackage/test/completer/handler.spec.ts

```diff
@@ -47,7 +47,7 @@ class TestCompleterModel extends CompleterModel {
 class TestCompletionHandler extends CompletionHandler {
   methods: string[] = [];
 
-  onTextChanged(str: ISharedText, changed: SourceChange): void {
+  async onTextChanged(str: ISharedText, changed: SourceChange): Promise<void> {
```

Comment:
@krassowski do you know why the completer tests are in the `metapackage` and not in the `completer` package?

Reply:
These were moved there in #12495 by @fcollonval as far as I understand to remove a dependency cycle, but I don't know the details and whether this is still needed.

Reply:
Yes lerna does not distinguish between dep and dev dep when building the packages dep graph.

And for tests, they were chicken/egg issue; like notebook depends on the completer but the completer tests depends on notebook (unit tests should have used mocked documents for interface - but better integration tests than nothing). Hence the need to move the tests to an upper package.